src/main/java/org/jabref/gui/groups/GroupTree.css

```diff
@@ -1,10 +1,10 @@
 .tree-table-view, .tree-table-row-cell {
-    -fx-background-color: #dadad8;
+    -fx-background-color: -fx-control-inner-background;
     -fx-table-cell-border-color: transparent; /* hide grid lines */
 }
 
 .tree-table-cell {
-    -fx-font-size: 105%;
+    /*-fx-font-size: 105%;*/
```

Comment:
That's probably personal preference, but I prefer the bigger font size (and smaller for hits).

Reply:
I'm against this. Although the font is only one pixel larger on my system and the anti-aliasing is also OK, it introduces a very notable font-difference to the main-table where there should be consistency.  I would accept if this is wanted by most devs, but from my point of view, it is not OK.

Reply:
Ok, convinced. The base font size should be the same. But I still think the hit-number can be smaller because this is kind of unimportant info.

By the way, did you change the padding of the tree node items? They have a bit tighter feeling than on the current master/maintable-beta, which probably results not from the difference of font-sizes.